src/test/java/com/singlestore/jdbc/integration/DatabaseMetadataTest.java

```diff
@@ -503,7 +503,11 @@ public void testGetColumns() throws SQLException {
     assertEquals("", rs.getString(12)); // REMARKS
 
     assertTrue("null".equalsIgnoreCase(rs.getString(13)) || rs.getString(13) == null); // COLUMN_DEF
-    assertEquals(32 * 3, rs.getInt(16)); // CHAR_OCTET_LENGTH
+    if (minVersion(8, 7, 0)) {
```

Comment:
We can but I prefer leaving it as is.
I caught a bug that was visible only when the charset was different.

Reply:
sure, testing with different charsets is necessary. I meant that it would be good to test metadata different charsets on the same server version, but this would better be done as an additional task.

Reply:
I see
I created a ticket for it https://memsql.atlassian.net/browse/PLAT-7195